source/server/spongineer/incompatible.rst

```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
+- Solution: None so far
+- Github issue: `Logged slot transactions without event <https://github.com/SpongePowered/Sponge/issues/3680>`_
+
+Pehkui
+~~~~~~
+
+- Version: 3.4.2
 - Problem: Crash on startup
-- Solution: Change ``B:useconfigs=false`` to ``true`` in the ``config/mystcraft/core.cfg`` file
-
-Open Terrain Generator (OTG)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-- Dependencies: OTGcore
-- Versions: All (so far)
-- Problem: Crash on startup / Multiworld Wgen problems
-- Solution: Pre-generate world without SpongeForge, then remove OTG and add SpongeForge.
-
-Phosphor
-~~~~~~~~
-- Versions: Up to 0.2.4
-- Problem (1): Crash on startup
-- Solution: Update to version 0.2.5 or newer, which is compatible with Sponge RC3844.
-- Problem (2): Poor graphic performance
-- Solution: Set the optimisation ``async-lighting`` to ``false`` in the Sponge ``global.conf`` file.
-
-There may be many more, please help us keep this list maintained by contributing to the SpongeDocs on GitHub!
-The :doc:`debugging` page may also be of help if your issue is not one of those mentioned above.
+- Solution: None so far
+- Github issue: `SpongeForge incompatibility with pehkui <https://github.com/SpongePowered/Sponge/issues/3829>`_
+
+Physics Mod
+~~~~~~~~~~~
+
+- Version: 1.3.4
+- Problem: Lag on TNT use 
+- Solution: None so far
+- Github issue: `Fatel server lag when using TNT and Physics mod <https://github.com/SpongePowered/Sponge/issues/3517>`_
+
+Progressive Bosses
+~~~~~~~~~~~~~~~~~~
+
+- Version 3.4.3
+- Problem: Crash on startup
+- Solution: Update SpongeForge to the latest
+- Github issue: `Illegal State Exception with Progressive Bosses mod <https://github.com/SpongePowered/Sponge/issues/3714>`_
+
+Random patches
+~~~~~~~~~~~~~~
+
+- Version: 2.4.4
+- Problem: Crash on startup
+- Solution: Update SpongeForge to the latest
+- Github issue: `Server crash with Randompatches <https://github.com/SpongePowered/Sponge/issues/3589>`_
+
+Tickers' Construct
+~~~~~~~~~~~~~~~~~~
+
+- Version: 1.16.5-3.1.2.265
+- Problem: Non-fatal console error when opening an inventory
+- Solution: None so far
+- Github issue: `Tinkers' Construct slot transaction spam with SpongeForge <https://github.com/SpongePowered/Sponge/issues/3527>`_
+
+Twilight Forest
+~~~~~~~~~~~~~~~
+
+- Version: 4.0.870
+- Problem: Crash on startup
+- Solution: Update SpongeForge to the latest
+- Github issue: `SpongeForge has fatal errors with Twilight Forest mod <https://github.com/SpongePowered/Sponge/issues/3574>`_
+
+Valkyrien Skies
+~~~~~~~~~~~~~~~
+
+- Version: 116-2.0.0-alpha6
+- Problem: Crash on startup
+- Solution: None so far
+- Github issue: `The server can't launch with Valkyrien Skies <https://github.com/SpongePowered/Sponge/issues/3809>`_
+
+World Edit
+~~~~~~~~~~
+
+- Version: 7.2.5
+- Problem: Fatal errors after startup relating to commands
+- Solution: None so far
+- Github issue: `WorldEdit command registrar issue on server start <https://github.com/SpongePowered/Sponge/issues/3540>`_
```

Comment:
There's also this issue, covering the issue with the Sponge version. Progress is happening...
https://github.com/SpongePowered/Sponge/issues/3811

Reply:
Added issue.

Reply:
I have heard the plugin version is now working on Sponge Vanilla, does the plugin also work on sponge forge too?

Reply:
I haven't seen (3) and (4) from issue 3811 fixed yet, so probably not.